views/error404.html

```diff
@@ -0,0 +1,17 @@
+% include('base.html', title="PyJudge")
+
+<body class="h-100">
+  <div class="row h-100 align-items-center justify-content-center text-center" style="min-height:600px">
+    <div class="container" >
+      <h1 class="display-1">{{errorcode}}</h1>
+  		<h4 class="text-muted"> {{errorbody}}</h4>
```

Comment:
as we don't have css now, `display-1` class can be removed.
error body can be written in a `p` tag
sizing using `h-100` and `min-height` can be removed (not necessary)

Reply:
this all are bootstrap predefined classes for resizing the text

Reply:
min-height is used for aligned the message in vertical center other it will be on the top